designcompose/src/main/java/com/android/designcompose/InteractionState.kt

```diff
@@ -588,6 +589,15 @@ private fun searchNodes(
             }
         }
     }
+
+    if (q is NodeQuery.NodeId) {
+        // Look for the view in our map of views indexed by node ID.
+        val view = nodeIdMap[q.value]
```

Comment:
nit: I think since this function can return null, it would be OK to directly return `nodeIdMap[q.value]`.

Reply:
Done.